sos/report/plugins/juju.py

```diff
@@ -33,26 +62,99 @@ def setup(self):
         self.add_copy_spec("/var/lib/juju/agents/*/agent.conf")
 
         # Get a directory listing of /var/log/juju and /var/lib/juju
-        self.add_dir_listing([
-            '/var/log/juju*',
-            '/var/lib/juju*'
-        ], recursive=True)
+        self.add_dir_listing(
+            ["/var/log/juju*", "/var/lib/juju*"], recursive=True
+        )
```

Comment:
I don't think we need to change this here, and leave this as is. With our project, we prefer the format with `([` being together as it was before